test/functionalTest/cases/0876_entity_dates/filter_datecreated_sub.test

```diff
@@ -26,7 +26,7 @@ Filter with dateCreated in subscriptions
 --SHELL-INIT--
 dbInit CB
 brokerStart CB
-accumulatorStart
+accumulatorStart --pretty-print
```

Comment:
The "ugly" thing is having in a line one style and the other in the next one, eg:

brokerStart CB 0-255)
accumulatorStart --pretty-print

That uglyness doesn't happen with orionCurl due to, in the SHELL part, we are not mixing style.

Another advantage of having a different style is that is easy to make clear it is an arguments for the wrapper (e.g. `0-255`) that are not exactly the same than the ones that the wrapper will use to run the actual command (e.g. `-t 0-255`). That difference could help to debug the test harness suite in the cases of problems.

However, not so important at the present moment. It could be deferred from this PR and fixed (in a way or other) when that ugliness becomes unbearable. As you prefer.